packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx

```diff
@@ -187,105 +169,106 @@ html {margin: 0; display: grid; min-height: 100%}
     border-bottom-width: 1px;
     border-left-width: 1px;
     -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale
+    -moz-osx-font-smoothing: grayscale;
+    margin: 0
   }
-  div:where([data-ws-component="Box"]) {
+  :where(div.w-box) {
     box-sizing: border-box;
     border-top-width: 1px;
     border-right-width: 1px;
     border-bottom-width: 1px;
     border-left-width: 1px;
     outline-width: 1px
   }
-  address:where([data-ws-component="Box"]) {
+  :where(address.w-box) {
     box-sizing: border-box;
     border-top-width: 1px;
     border-right-width: 1px;
     border-bottom-width: 1px;
     border-left-width: 1px;
     outline-width: 1px
   }
-  article:where([data-ws-component="Box"]) {
+  :where(article.w-box) {
```

Comment:
do you think it makes sense to just have a simple .w-box here?

Reply:
Nope. We use :where to reset specificity and let stateless styles override hover. Same like we do with a:visited {} styles.

Reply:
Same effect would give be using `@layer preset {}`

Reply:
right, forgot about it

Reply:
ok but why do we need the tag selector as well?